src/lib/runtime_config/runtime_config.ml

```diff
@@ -1802,14 +1791,8 @@ module Json_loader : Json_loader_intf = struct
         | Ok loaded_config ->
             combine config loaded_config
         | Error err ->
-            [%log fatal]
```

Comment:
This means that we won't see the failure reason in the node logs, and so we'll have a harder time root-causing crashes if node operators hit this :(

Reply:
for this specific log, won't node operators just see the error in the exception back trace?